playground/src/components/submission_list.tsx

```diff
@@ -1,56 +1,50 @@
-import type { Exercise } from "@/model/exercise";
+import type { Submission } from "@/model/submission";
 import type { Feedback } from "@/model/feedback";
 
-import useSubmissions from "@/hooks/playground/submissions";
-
 import Disclosure from "@/components/disclosure";
 import SubmissionDetail from "@/components/details/submission_detail";
 
 type SubmissionListProps = {
-  exercise: Exercise;
+  submissions: Submission[];
   feedbacks?: Feedback[];
   onFeedbacksChange?: (feedback: Feedback[]) => void;
 };
 
-export default function SubmissionList({ exercise, feedbacks, onFeedbacksChange }: SubmissionListProps) {
-  const { data, isLoading, error } = useSubmissions(exercise);
-
-  if (data) {
-    let feedbacksBySubmissionId: Record<number, Feedback[]> = {};
-    if (feedbacks) {
-      feedbacksBySubmissionId = feedbacks.reduce((acc, feedback) => {
-        if (!acc[feedback.submission_id]) {
-          acc[feedback.submission_id] = [];
-        }
-        acc[feedback.submission_id].push(feedback);
-        return acc;
-      }, {} as Record<number, Feedback[]>);
-    }
-
-    return (
-      <Disclosure
-        title={`${data.length} Submission${
-          data.length === 1 ? "" : "s"
-        }`}
-        className={{ content: "space-y-1" }}
-      >
-        {data.map((submission) => (
-          <Disclosure title={`Submission ${submission.id}`} key={submission.id}>
-            <SubmissionDetail
-              key={submission.id}
-              submission={submission}
-              feedbacks={feedbacksBySubmissionId[submission.id]}
-              onFeedbacksChange={onFeedbacksChange}
-            />
-          </Disclosure>
-        ))}
-      </Disclosure>
-    );
-  } else if (error) {
-    return <div className="text-gray-500 text-sm">Failed to load submissions</div>;
-  } else if (isLoading) {
-    return <div className="text-gray-500 text-sm">Loading submissions...</div>;
-  } else {
-    return null;
+export default function SubmissionList({
+  submissions,
+  feedbacks,
+  onFeedbacksChange,
+}: SubmissionListProps) {
+  let feedbacksBySubmissionId: Record<number, Feedback[]> = {};
+  if (feedbacks) {
+    feedbacksBySubmissionId = feedbacks.reduce((acc, feedback) => {
+      if (!acc[feedback.submission_id]) {
+        acc[feedback.submission_id] = [];
+      }
+      acc[feedback.submission_id].push(feedback);
+      return acc;
+    }, {} as Record<number, Feedback[]>);
   }
+  
+  if (submissions.length === 0) {
+    return <div className="text-gray-500 text-sm">No submissions</div>;
+  }
+
+  return (
+    <Disclosure
+      title={`${submissions.length} Submission${submissions.length === 1 ? "" : "s"}`}
+      className={{ content: "space-y-1" }}
```

Comment:
What is this? I don't know this syntax for react 🤔 
```suggestion
      className="space-y-1"
```

Reply:
I guess I'm abusing the prop name a bit? `Disclosure` has this prop type:
```ts
className?: {
    root?: string,
    trigger?: string,
    content?: string,
  }
```